composer.json

```diff
@@ -38,21 +37,16 @@
     },
     "suggest": {
         "ext-curl": "If you use CURLRequest class",
-        "ext-dom": "If you use TestResponse",
         "ext-exif": "If you run Image class tests",
-        "ext-fileinfo": "Improves mime type detection for files",
         "ext-gd": "If you use Image class GDHandler",
         "ext-imagick": "If you use Image class ImageMagickHandler",
-        "ext-libxml": "If you use TestResponse",
```

Comment:
No.

> This extension requires [» libxml](http://www.xmlsoft.org/) >= 2.9.0 as of PHP 8.0. libxml >= 2.6.0 prior to PHP 8.0.
https://www.php.net/manual/en/libxml.requirements.php

Reply:
It's true.
However PHP handles downstream dependencies for you. There is no need to check for the required libraries downstream. 

If PHP states 

> The libxml extension is enabled by default,

 https://www.php.net/manual/en/libxml.installation.php, that's all there is to check.

Let's dig deeper:
```
php=$(which php)
php_pkg=$(rpm -qf $php) 
rpm -q "${php_pkg}" --requires
```
, or
`rpm -q "${php_pkg}" --requires | grep -i xml`


Finally:
(there is no package for libxml)
`dnf search libxml | grep -i php`
(included by default)
`php -r "phpinfo();" | grep -i xml`

Cannot check the same on Windows and others, but it should be similar.
So the remarks regarding http://www.xmlsoft.org/ do not apply unless there is a different approach from PHP on other operating systems. I'm using Fedora.